services/121-service/src/programs/program/dto/custom-data.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiModelProperty } from "@nestjs/swagger";
+
+import { Length, IsString, IsNotEmpty } from "class-validator";
+
+export class CustomDataDto {
+  @ApiModelProperty({ example: 'did:sov:2wJPyULfLLnYTEFYzByfUR' })
+  @Length(29, 30)
+  public readonly did: string;
+  @ApiModelProperty({ example: 'phone' })
+  @IsNotEmpty()
+  @IsString()
+  public readonly key: string;
+  @ApiModelProperty({ example: 'BSN:244672027' })
```

Comment:
I would suggest to use the same example as the government does(https://www.rijksoverheid.nl/onderwerpen/privacy-en-persoonsgegevens/vraag-en-antwoord/waar-vind-ik-mijn-burgerservicenummer-bsn-op-mijn-paspoort)

```suggestion
  @ApiModelProperty({ example: 'NL:999999990' })
```